package.json

```diff
@@ -8,14 +8,14 @@
   },
```

Comment:
Note to reviewers:

I did _not_ add ` --snow=false` to the `build:test:dev:mv3` script because I don't understand that script. It sets the task/`buildTarget` to `testDev` and sets `--apply-lavamoat=false`, but the MV3 build system running with the `TEST_DEV` `buildTarget` forces lavamoat anyway (but in "stats mode").

see (where `buildTarget === testDev`):

0. https://github.com/MetaMask/metamask-extension/blob/f2be12cc8a5057dde6add6534ac6933a46c310ee/development/build/constants.js#L12
0. https://github.com/MetaMask/metamask-extension/blob/f2be12cc8a5057dde6add6534ac6933a46c310ee/development/build/utils.js#L28-L38
1. https://github.com/MetaMask/metamask-extension/blob/f2be12cc8a5057dde6add6534ac6933a46c310ee/development/build/scripts.js#L1175-L1178
2. https://github.com/MetaMask/metamask-extension/blob/f2be12cc8a5057dde6add6534ac6933a46c310ee/app/scripts/app-init.js#L8
3. https://github.com/MetaMask/metamask-extension/blob/f2be12cc8a5057dde6add6534ac6933a46c310ee/app/scripts/app-init.js#L71-L76
4. https://github.com/MetaMask/metamask-extension/blob/f2be12cc8a5057dde6add6534ac6933a46c310ee/development/build/scripts.js#L541-L547

It actually more like a `start:` script than a `build:` script, so that makes it even more confusing.

Since this isn't used in CI I stopped investigating why it is the way it is and am writing this comment instead.